receiver/hostmetricsreceiver/README.md

```diff
@@ -64,6 +64,15 @@ filesystem:
     match_type: <strict|regexp>
 ```
 
+### Load
+
+`per_cpu` divides the average load per CPU (default: `false`).
+
+```yaml
+load:
+  per_cpu: <false|true>
```

Comment:
This is confusing since it does not mean the same thing as "per_cpu" in the cpu metrics.

Also please ensure we are in sync with the specification, see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md